docs/integrate_tensorflow_datasets.md

```diff
@@ -0,0 +1,17 @@
+# Instructions to Integrate TFDS Datasets
+
+1. Get the name, version number of the Tensorflow Dataset, and optionally the config: "name[/config]:version_number", where the brackets denote optional text.
+2. Set the environmental variables ARMORY_PRIVATE_S3_ID and ARMORY_PRIVATE_S3_KEY to the appropriate keys with write access to the Armory S3 bucket. 
+3. From a locally cloned version of armory, on a new branch, run:
+```
+python -m armory exec pytorch -- python -m armory.data.integrate_tfds name[/config]:version
+```
+where the brackets denote optional text.
+
+4. The script will download and process the TFDS dataset, generate TF Records files, create a tarball, and upload the tarball to S3. It also will create a S3 checksum file in ```armory/data/cached_s3_checksums/{name}.txt```
```

Comment:
Very minor, but step 4 isn't really a step (in the sense that it requires user action). I'd leave the text but attach it to step 3 which will make clear that it's describing what that command does